<launch>
    <node pkg="n_cpp" type="tester" name="tester" output="screen" launch-prefix="gdb -ex run --args"/>
</launch>
